scripts/dss-start-stop--think-before-you-invoke-me.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+"""
+Use the AWS Lambda concurrency property to disable/enable lambda functions:
+    - set concurrency to 0 to disable execution
+    - remove concurrency setting to enable execution
+"""
+import os
+import sys
+import boto3
+import click
+import argparse
+
+
+stage = os.environ['DSS_DEPLOYMENT_STAGE']
```

Comment:
Shouldn't this be further down in the code.=?

Reply:
It will fail immediately if this environment variable is not set.

Reply:
OK